addon/services/syncer.js

```diff
@@ -385,6 +453,14 @@ export default Ember.Service.extend({
 
     return this._changesForRecord(store, job).then((changes) => {
       record.setProperties(changes);
+      return RSVP.Promise.resolve(record);
```

Comment:
Возвращаемое из `then` значение не обязательно заворачивать в промис.
```suggestion
      return record;
```